docs/conceptual/using-gpu-sanitizer.md

```diff
@@ -90,7 +91,7 @@ If it does not appear, when executed the application will quickly output an ASan
 
 * Ensure that the application `llvm-symbolizer` can be executed, and that it is located in `/opt/rocm-<version>/llvm/bin`. This executable is not strictly required, but if found is used to translate ("symbolize") a host-side instruction address into a more useful function name, file name, and line number (assuming the application has been built to include debug information).
 
-There is an environment variable, `ASAN_OPTIONS`, that can be used to adjust the runtime behavior of the ASAN runtime itself. There are more than a hundred "flags" that can be adjusted (see an old list at [flags](https://github.com/google/sanitizers/wiki/AddressSanitizerFlags)) but the default settings are correct and should be used in most cases. It must be noted that these options only affect the host ASAN runtime. The device runtime only currently supports the default settings for the few relevant options.
+There is an environment variable, `ASAN_OPTIONS`, that can be used to adjust the runtime behavior of the Asan runtime itself. There are more than a hundred "flags" that can be adjusted (see an old list at [flags](https://github.com/google/sanitizers/wiki/AddressSanitizerFlags)) but the default settings are correct and should be used in most cases. It must be noted that these options only affect the host Asan runtime. The device runtime only currently supports the default settings for the few relevant options.
```

Comment:
The sanitizers in abbreviated form use a camelcase ( e.g., ASan, LSan, etc. ; see usage on https://clang.llvm.org/docs/AddressSanitizer.html , https://clang.llvm.org/docs/LeakSanitizer.html ). That being said, ASan is often written as ASAN, asan, ASan and so being consistent is just as important as "correct" as "correct" seems somewhat in the eye of the beholder

Reply:
We have used ASan in text and ASAN in code snippets previously.

Reply:
I will leave the environment variable as "ASAN_OPTIONS"

Reply:
Fixed the ASan LSan issues.